src/lsm/forest_fuzz.zig

```diff
@@ -370,8 +498,13 @@ pub fn generate_fuzz_ops(random: std.rand.Random, fuzz_op_count: usize) ![]const
                     // Can only checkpoint on the last beat of the bar.
                     compact_op % constants.lsm_batch_multiple == constants.lsm_batch_multiple - 1 and
                     compact_op > constants.lsm_batch_multiple and
+                    // Never checkpoint at the same op twice
+                    compact_op > checkpointed_op + constants.lsm_batch_multiple and
                     // Checkpoint at roughly the same rate as log wraparound.
                     random.uintLessThan(usize, Environment.compacts_per_checkpoint) == 0;
+                if (checkpoint) {
+                    checkpointed_op = op - constants.lsm_batch_multiple;
```

Comment:
Is this reachable when `op == constants.lsm_batch_multiple - 1`?

Reply:
I guess only reachable with the change I suggested above. In which case changing `-` to `-|` might be an easy fix.

Reply:
Even with the change you suggested it's still unreachable because one of the other conjuncts is `compact_op > constants.lsm_batch_multiple`.